docs/rules/idiomatic/use-in-operator.md

```diff
@@ -2,7 +2,7 @@
 
 **Summary**: Use `in` to check for membership
 
-**Category**: Style
```

Comment:
Considering backwards compatibility for this one, and apologies for not having done so in the issue 😅 But I think it's probably good to keep the` docs/rules/style/use-in-operator.md` file around for a while but have it contain only a link to the doc in the idiomatic category. That way, users of older versions would still find the docs and be able to update their config if needed. Does that make sense?

Reply:
Ah yeah that makes sense now I think so basically two readme's but the one in `docs/rules/style/use-in-operator.md` just points to the "new" one (of course nicely) in `docs/rules/idiomatic/use-in-operator.md` am I following you on this? 😅

Reply:
Yep! We could just say something like "Rule moved to idiomatic category" and provide a link to its new location.

Reply:
Hey! 👋 

LMK what you think, happy to change it a bit more if needed 👍 . I added back the style reference guide and added a note to the old docs pointing to the new ones.